src/encode.ts

```diff
@@ -43,11 +43,11 @@ const defaultEncodeOptions: EncodeOptions = {};
  *
  * The returned buffer is a slice of a larger `ArrayBuffer`, so you have to use its `#byteOffset` and `#byteLength` in order to convert it to another typed arrays including NodeJS `Buffer`.
  */
-export function encode<ContextType>(
+export function encode<ContextType = undefined>(
```

Comment:
Unless giving `undefined` as default type, the following code will cause type mismatch errors:

```
encode(someValue, { sortKeys: true })
```

Even though this error can be avoidable by giving explicit type annotation to the code above like `encode(someValue, { sortKeys: true } as EncodeOptions)`, this will be braking change of the API.